### PR TITLE
Prac 5 code review request

### PR DESCRIPTION
**Pull request by PhillipCluff**
Please check formatting, naming and logic @zackaryjwright-jcu @martinarmstrongAU

### CODE REVIEW

File: prac_05/state_names.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+state = input("Enter short state: ")
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
```
Comment (zackaryjwright-jcu) [inline prac_05/state_names.py:15]: should you use the new print style and use the .format

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (MartinArmstrongAU) [inline .gitignore:1]: Why is this added into the branch?
Prac_05 should be a new directory, not a new project.
Hence, .gitignore shouldn't be needed here.

**Review by MartinArmstrongAU — COMMENTED** [review]
Check your commit message before committing so that there are no typos.

File: prac_05/state_names.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+State names in a dictionary
+File needs reformatting
+"""
+
+
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(STATE_NAMES)
+
+state = input("Enter short state: ")
```
Comment (MartinArmstrongAU) [inline prac_05/state_names.py]: The input should handle lower case values by using the .upper() function.

File: prac_05/state_names.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+while state != "":
+    if state in STATE_NAMES:
+        print(state, "is", STATE_NAMES[state])
+    else:
+        print("Invalid short state")
+    state = input("Enter short state: ")
```
Comment (MartinArmstrongAU) [inline prac_05/state_names.py]: Same again for handling input upper/lower case

**Review by MartinArmstrongAU — COMMENTED** [review]
Your commit message should include everything you commit.  All you have said is "add word_occurrences" - yet, quick_picks, hex_colours and state_names are all added in this commit.
